sympy/physics/units/definitions/unit_definitions.py

```diff
@@ -173,15 +173,15 @@
 
 ha = hectare = Quantity("hectare", abbrev="ha")
 
-l = L = liter = liters = Quantity("liter")
+l = L = liter = liters = Quantity("liter", abbrev="l")
```

Comment:
I would suggest discussing, if it should be `liter` or `litre` for the sake of correctness although it is a minor change. This is what I found regarding litre being the international standard - https://en.wikipedia.org/wiki/Litre#:~:text=The%20spelling%20used%20by%20the,predominantly%20used%20in%20American%20English.